sdk/appservice/arm-appservice/src/webSiteManagementClientContext.ts

```diff
@@ -13,11 +13,12 @@ import * as msRest from "@azure/ms-rest-js";
 import * as msRestAzure from "@azure/ms-rest-azure-js";
 
 const packageName = "@azure/arm-appservice";
-const packageVersion = "5.8.0";
+const packageVersion = "5.9.0";
```

Comment:
version here also need to be upgrade to a major bump due to breaking changes

Reply:
Merge it